Source/AliveLibAE/Input.cpp

```diff
@@ -123,7 +132,7 @@ ALIVE_ARY(1, 0x55EAD8, InputBinding, 36, sDefaultKeyBindings_55EAD8, {
     { VK_NUMPAD7, InputCommands::eGameSpeak7 },
     { VK_NUMPAD8, InputCommands::eGameSpeak8 },
     { VK_NUMPAD0, InputCommands::eChant },
-    { 'C', e0x80000000 },
+    { 'C', eConfigure },
```

Comment:
should probably fully qualify like the others InputCommands::eConfigure  ?

Reply:
Yup, missed that one, thanks!